spec/dummy/app/models/division.rb

```diff
@@ -3,6 +3,9 @@ class Division < ApplicationRecord
   include Hashable
 
   belongs_to :owner, class_name: 'User'
+  belongs_to :category, optional: true
+
+  attr_accessor :category_name
```

Comment:
@calebtocco I wish there was a way we could do this without using an attr_accessor and dynamically add it or move it into a reusable concern on the controller or something like that, I prefer less wiring up if possible, but I think its good for now since that would probably be a lot more work maybe thats a phase 2 thing.

Reply:
right, we should be able to improve on this in the future